### DISCUSSION TITLE
Image Gallery: Make it possible with a File Grid field to show only the image files and be able to sort them through dragging

### DISCUSSION THREAD

**Discussion by strudelmedia** [ExpressionEngine/ExpressionEngine#4188] [Feature Requests]
### Suggest an idea for ExpressionEngine

**Is your feature request related to a problem? Please describe.**
A client wants a photo gallery that they could order and reorder through dragging while just focusing on the images. With a File Grid, re-ordering is really non-visual, since you can't actually see the images that you're moving. The way it is now, if you have an image, a photo credit, a caption, info about the image, and then further info about how to deal with the image on the page, you can't re-sort images visually.

**Describe the solution you'd like**
The way I would like to see this work is that for ordering purposes, I could suppress all the other fields in the File Grid and see a grid of images that I could drag and drop to reorder they way I would like. The idea is to reorder images in a visual way.

**Describe alternatives you've considered**

I haven't found any alternatives — Channel Images had a feature sort of like this, but a user couldn't easily switch back and forth from that view. Also, I think it would be much better to have a native solution.

**Teachability, Documentation, Adoption, Migration Strategy**
If you can, explain how users will be able to use this and possibly write out a version the docs. Some of the best features are born from writing documentation first. 

Ideally, there would be switch that end users could use that would change the view from columns/rows to a grid view that could be sorted by dragging/dropping.

- [ ] I am capable and would like to work on implementation of this feature if it is considered.

**Comment by intoeetive** [comment]
@TomJaeger @bryannielsen @Yulyaswan this is related to the discussion we had some time ago: https://packettide.slack.com/archives/C030Y8HG9L3/p1708009415279859

**Comment by strudelmedia** [comment]
Hi!

Thanks for letting me know about this — so I can’t actually view that link because although I’m on Slack I can’t access that one. Could you tell what the discussion was a bit?

Thank you!

Edward

> On Apr 1, 2024, at 9:03 AM, Yuri Salimovskiy ***@***.***> wrote:
> 
> 
> @TomJaeger @bryannielsen @Yulyaswan this is related to the discussion we had some time ago: https://packettide.slack.com/archives/C030Y8HG9L3/p1708009415279859
> —
> Reply to this email directly, view it on GitHub, or unsubscribe.
> You are receiving this because you authored the thread.Message ID: ***@***.***>




Edward

…………………………………………
edward ratliff / strudelmedia design
strudelmediadesign.com
tel 646 666 9595

++want to send me a file?
go to strudelmedia.wetransfer.com

**Reply by intoeetive** [reply]
@strudelmedia - sorry, that's more of our internal note. There's not much to see there anyway, but we were talking about expanding File Grid to serve better for "Image Gallery" use case

**Comment by strudelmedia** [comment]
Ah, okay! Thanks.

> On Apr 1, 2024, at 9:43 AM, Yuri Salimovskiy ***@***.***> wrote:
> 
> 
> @strudelmedia - sorry, that's more of our internal note. There's not much to see there anyway, but we were talking about expanding File Grid to serve better for "Image Gallery" use case
> —
> Reply to this email directly, view it on GitHub, or unsubscribe.
> You are receiving this because you were mentioned.Message ID: ***@***.***>




Edward

…………………………………………
edward ratliff / strudelmedia design
strudelmediadesign.com
tel 646 666 9595

++want to send me a file?
go to strudelmedia.wetransfer.com
